config/management/operational/src/validator_set.rs

```diff
@@ -43,8 +43,7 @@ pub async fn decode_validator_set(
         let config = DecodedValidatorConfig::from_validator_config(info.config())
             .map_err(|e| Error::NetworkAddressDecodeError(e.to_string()))?;
 
-        let config_resource = client.validator_config(*info.account_address()).await?;
-        let name = DecodedValidatorConfig::human_name(&config_resource.human_name);
+        let name = String::from("");
```

Comment:
why delete that component?

Reply:
the name field? I don't have strong opinion on this, we can add it back